cfgov/sheerlike/templates.py

```diff
@@ -8,8 +8,6 @@ def _convert_date(date, tz):
         if isinstance(date, basestring):
             date = parser.parse(date,
                                 default=datetime.datetime.today().replace(day=1))
-        # TODO: Event times are currently stored as UTC in the database, but they're actually ET times. 
-        # We will need to fix this once Wagtail fixes its handling of TIME_ZONE. 
         if type(date) in [datetime.datetime, datetime.date]:
             if isinstance(date, datetime.datetime):
                 pytzone = timezone(tz)
```

Comment:
Don't we also need to update this function so that it's no longer stripping the timezone info at line 17? `date = date.replace(tzinfo=None)`